spec/lib/salus/scanners/report_go_dep_spec.rb

```diff
@@ -48,6 +48,72 @@
     end
   end
 
+  describe '#record_dep_from_go_sum' do
+    it 'should report on all the dependencies in the go.sum file' do
+      repo = Salus::Repo.new('spec/fixtures/report_go_sum')
+      scanner = Salus::Scanners::ReportGoDep.new(repository: repo, config: {})
+
+      scanner.run
+
+      dependencies = scanner.report.to_h.fetch(:info).fetch(:dependencies)
+
+      expect(dependencies[0..2]).to match_array(
+        [
+          {
+            dependency_file: 'go.sum',
+            type: 'golang',
+            name: 'github.cbhq.net/c3/bls12-381',
+            reference: 'N/A for go.mod/go.sum dependencies',
+            version_tag: "v0.0.0-20210114210818-577bfdc5cb9c"
+          },
+          {
+            dependency_file: 'go.sum',
+            type: 'golang',
+            name: 'github.cbhq.net/c3/bls12-381',
+            reference: 'N/A for go.mod/go.sum dependencies',
+            version_tag: "v0.0.0-20210114210818-577bfdc5cb9c/go.mod"
```

Comment:
The /go.mod should probably be dropped from the version here.  I think we'll want to abstract the go.sum formats away from the output here.